pom.xml

```diff
@@ -101,12 +101,11 @@
 			<groupId>org.javassist</groupId>
 			<artifactId>javassist</artifactId>
 		</dependency>
-
 		<dependency>
 			<groupId>org.scijava</groupId>
 			<artifactId>scijava-common</artifactId>
-			<scope>test</scope>
```

Comment:
Oof, I didn't realize that ij1-patcher doesn't depend on scijava-common. Nor should it, I don't think.

So then: we will need to replicate the logic of `Context.getClassLoader()`, rather than being able to call it directly. :frowning: It's not so bad—just a null check on the context class loader result, and then `ClassLoader.getSystemClassLoader()` if null.

Reply:
Fixed in 429b8c9. I'd vote for a squash and merge at the end of the day.